src/runtime_src/core/pcie/driver/linux/xocl/subdev/mailbox.c

```diff
@@ -1222,7 +1296,7 @@ static bool do_hw_rx(struct mailbox_channel *ch)
 		return true;
 	case PKT_MSG_START:
 		if (ch->mbc_cur_msg) {
-			MBX_ERR(mbx, "Received partial msg (id 0x%llx)\n",
+			MBX_DBG(mbx, "Received partial msg (id 0x%llx)\n",
```

Comment:
Why not print it out by default? if this shows up in your fuzz test too often, maybe we should consider treating it as malicious guest after seeing it xx times and turn off the mailbox?

Reply:
flood of err msg will be on console and makes the host slow. changed to MBX_WARN